eksupgrade/cli.py

```diff
@@ -103,5 +79,15 @@ def entry(args: Optional[List[str]] = None) -> None:
     main(parsed_arguments)
 
 
+def get_eks_supported_regions() -> List[str]:
```

Comment:
I thought about doing this myself when I expanded the list, but I am not sure we even want this region enforcement.  We don't actually use the region anywhere except maybe the unused eksctl module (which should likely be removed).

@bryantbiggs Thoughts?  Should we simply eliminate this all together?

Reply:
ya, I don't see a valid use case where we need to know the supported regions. Users are going to tell us their cluster name and the region its located in - if they do either of those incorrectly, its just incorrect and they will need to re-provide the correct information

If we were trying to create a new cluster from scratch - then supported regions would have a valid use case (maybe, the API would still return an error if the region is not supported)

Reply:
That makes sense from the api call and plan forward. We can close the issue # [25](https://github.com/aws-samples/amazon-eks-one-click-cluster-upgrade/issues/25) and this PR then.

Reply:
Thank you!